github_pr.py

```diff
@@ -149,6 +149,40 @@ def _return_specific_comment_prs(all_prs, filters):
         return []
 
 
+def _check_approved_mergers_file(approved_users, comment_users):
+    if not isinstance(approved_users, list):
```

Comment:
its a bit confusing this way
i was trying to reduce the amount of duplicate logic.
approved_users can be a list, from the option --mergecondition-approved-mergers
approved_users can be a string, from the option --mergecondition-approved-mergers-file and this passes the string into the open file option, then puts the contents of the file into the "approved_users" list.